### ISSUE TITLE
Atelencephalic microcephaly

### ISSUE THREAD

**Issue by patt1morgan** [obophenotype/human-phenotype-ontology#7206]
**Preferred term label:** Atelencephalic microcephaly

**Synonyms**

**Definition (free text, please give PubMed ID)**  Atelencephaly is a rare lethal congenital brain malformation characterized by underdevelopment of the prosencephalon and is often accompanied by the facial features seen in some cases of holoprosencephaly, such as cyclopia.     PMID: 26260203

**Parent term (use hpo.jax.org/app)** Primary microcephaly HP:0011451

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding term "Atelecephaly".
Atelencephalic microcephaly is best conceptualized as a disease.
